worlds/sc2/docs/custom_mission_orders_en.md

```diff
@@ -843,6 +853,99 @@ Grid supports the following index functions:
+- 'ggggg   bbbbb' # 7
+- 'gggg     bbbb' # 8
+- 'ggg  rrr  bbb' # 9
+- ' gg rrrrr bb ' # 10
+- '    rrrrr    ' # 11
+- '    rrrrr    ' # 12
+- '     rrr     ' # 13
+jump_distance_orthogonal: 2
+jump_distance_diagonal: 1
+missions:
+- index: group(g)
+  mission_pool: Terran Missions
+- index: group(b)
+  mission_pool: Protoss Missions
+- index: group(r)
+  mission_pool: Zerg Missions
+```
+This example draws the Archipelago logo using missions of different races as its colors. Note that while this example fits into 13 lines, there is no set limit for how many lines you may use, and likewise lines may be as long as you need them to be. Short lines are padded with spaces to match the longest line in the canvas, so lines are left-aligned in this case.
+
+You may have noticed that the above example has gaps between missions. Canvas layouts support jumping over gaps via `jump_distance_orthogonal` and `jump_distance_diagonal`, which determine the maximum distance over which two missions may be connected, in orthogonal and diagonal directions respectively. Missions at higher distances will only connect if there is no other mission in front of them.
+
+```yaml
+type: canvas
+canvas:
+- 'A  A'
+- 'B XB'
+jump_distance_orthogonal: 3
+jump_distance_diagonal: 0
+```
+In this example the two `A`s will connect because they are less than 3 missions apart, but the two `B`s will not connect because `X` is between them, and both `B`s will connect to `X` instead. Both sets of `AB`s will also connect because they are neighbors.
+
+Diagonal jumps function identically, with one exception:
+```yaml
+type: canvas
+canvas:
+- 'A  '
+- ' B '
+- ' XC'
+jump_distance_orthogonal: 1
+jump_distance_diagonal: 1
+```
+Missions that are diagonal neighbors only connect if they do not already share an orthogonal neighbor. In this example `A` and `B` connect, but `B` and `C` don't because `X` already connects them. No such restriction exists for higher-distance diagonal jumps, so it is recommended to keep `jump_distance_diagonal` low.
+
+Finally, the default entrance and exit on a canvas are dynamically set to be the non-empty slots that are closest to the top left and bottom right corner respectively. It is highly recommended to disable the defaults and set your own entrance and exit, for example like this:
+```yaml
+type: canvas
+canvas:
+- '  E  '
+- ' OOO '
+- ' O O '
+- ' OXO '
+missions:
+# Disable the defaults
+- index: entrances
+  entrance: false
+- index: exits
+  exit: false
+# Set our own
+- index: group(E)
+  entrance: true
+- index: group(X)
+  exit: true
+```
+
+#### Canvas Index Functions
+Canvas supports all of [Grid's index functions](#grid-index-functions), as well as the following:
+
+##### group(character)
+`group(character)` returns the indices which match the given character on the canvas. In the Archipelago logo example, `group(g)` gives the indices of all the `g`s on the canvas. Note that there is no group for spaces, so `group(" ")` does not work.
```

Comment:
Are there restrictions on the character? Will `group(.)` or `group(♥)` work? May be worth explicitly saying `group(\)` is disallowed for yaml reasons (can't expect all yaml writers to be programmers/know what escaping is).

Reply:
My concern is that `\` will behave differently depending on if it's wrapped in `"` or `'` quotes. Theoretically the quotes themselves also behave differently. This is for yaml reasons.

I suppose it's edge-case enough, especially if examples use single-quotes, that it's probably not worth mentioning.